autoload/lsp/diag.vim

```diff
@@ -134,7 +134,7 @@ def DiagsRefreshSigns(lspserver: dict<any>, bnr: number)
         prop_add(diag.range.start.line + 1,
                   util.GetLineByteFromPos(bnr, diag.range.start) + 1,
                   {end_lnum: diag.range.end.line + 1,
-                    end_col: util.GetLineByteFromPos(bnr, diag.range.end) + 1,
+                    end_col: util.GetLineByteFromPos(bnr, diag.range.end),
```

Comment:
This change breaks the highlight:

Before:

<img width="137" alt="Skærmbillede 2023-03-31 kl  10 55 49" src="https://user-images.githubusercontent.com/3134030/229074766-1b0a0fbd-d323-416f-b308-bc3017929638.png">


After:

<img width="145" alt="Skærmbillede 2023-03-31 kl  10 55 28" src="https://user-images.githubusercontent.com/3134030/229074678-5c2bb681-f401-4657-b734-06f6d58e8e3f.png">

Reply:
yea, had side effect if so.
i changed the way, now set a safeguard instead.